ClientApp/viking-vault/src/display_user_list/AddMoneyModal.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import './AddMoney.css';
+
+interface IAddMoneyModalProps{
+    open: boolean;
+    userName: string;
+    closeModal: any;
```

Comment:
You can specify function types, like this `func: (x: number, y: number) => number;`, you can use `void` return type if the function just does something